eo-maven-plugin/src/test/java/org/eolang/maven/FakeMaven.java

```diff
@@ -44,6 +45,7 @@
  * @since 0.28.12
  */
 @SuppressWarnings("PMD.TooManyMethods")
+@NotThreadSafe
```

Comment:
@volodya-lombrozo we normally place comment in JavaDoc `NOT thread-safe`.
If we are to use this new annotation we need to change it in all other places. I don't think it's reasonable in this case.

Reply:
@mximp `NOT thread-safe` is used only several times :) I've added `NOT thread-safe` to java doc. Please, check.